modules/swagger-codegen/src/main/resources/typescript-fetch/api.mustache

```diff
@@ -0,0 +1,309 @@
+// tslint:disable
+{{>licenseInfo}}
+
+import * as url                                 	    from "url";
+import * as isomorphicFetch                     	    from "isomorphic-fetch";
```

Comment:
`isomorphic-fetch` support run node.js and browser 

If browser does not Fetch API `isomorphic-fetch` use from [fetch polyfill](https://github.com/github/fetch)